### ISSUE TITLE
Merge file restore date not date of newest backup

### ISSUE THREAD

**Issue by schoenwaldnils** [AntonyCorbett/JWLMerge#64]
When merging two backups I sometimes have the restore date of the older backup when importing into JWLibrary

**Steps to reproduce the behavior:**
- Import two backups with different dates (one or more days apart) into JWLMerge
- create merge file
- import into JWLibray and see the restore date

**Expected behavior**
I wasn't entirely sure if this is a bug or a feature.
But in my opinion I'd expect the restore date of the newest of the backups

**Desktop:**
 - Windows version: Windows 10 Pro 20H2 19042.1466

**Comment by jayrtom** [comment]
Replicated also on windows 11. It happens everytime in my case

**Comment by schoenwaldnils** [comment]
The date of the original backup shows the date of the last modification of the personal study notes.
So I'd expect the date in the merged backup to be the date of the newest modification date across all files to merge 🤔

**Comment by schoenwaldnils** [comment]
Yes, I know what you mean. I'm just thinking when I create a backup in JWLibrary the last modified date is the date when I last changed a note or highlightes something. Not the date I created the backup 🤔 
Thats why I'd excpect to show the latest addition or change of the notes in the merged files 😁 

But I don't want to split hairs 😅 Main thing is that its not older that one of the backups that are merged
